### ISSUE TITLE
Explore keyboard focus indicator

### ISSUE THREAD

**Issue by rachaelbradley** [accessiblecommunity/accessiblecommunity.github.io#125]
The current focus indicator makes it difficult to read something in focus.  We could explore a color background change or solid outline instead of a dashed outline or adding some padding between the focus indicator and the text in focus. 

Good conversation to have with UX team.

**Comment by brian-montgomery** [comment]
@rachaelbradley, what is the standard for WCAG about the width/styling of focus indicators?

**Comment by rachaelbradley** [comment]
Success Criterion 2.4.13 Focus Appearance

[Understanding Focus Appearance](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html)
|
[How to Meet Focus Appearance](https://www.w3.org/WAI/WCAG22/quickref/#focus-appearance)
(Level AAA) [New]
When the keyboard [focus indicator](https://www.w3.org/TR/WCAG22/#dfn-focus-indicator) is visible, an area of the focus indicator meets all the following:

is at least as large as the area of a 2 [CSS pixel](https://www.w3.org/TR/WCAG22/#dfn-css-pixels) thick [perimeter](https://www.w3.org/TR/WCAG22/#dfn-perimeter) of the unfocused component or sub-component, and
has a contrast ratio of at least 3:1 between the same pixels in the focused and unfocused states.
Exceptions:

The focus indicator is determined by the [user agent](https://www.w3.org/TR/WCAG22/#dfn-user-agents) and cannot be adjusted by the author, or
The focus indicator and the indicator's background color are not modified by the author.
NOTE 1
What is perceived as the user interface component or sub-component (to determine enclosure or size) depends on its visual [presentation](https://www.w3.org/TR/WCAG22/#dfn-presentation). The visual presentation includes the component's visible [content](https://www.w3.org/TR/WCAG22/#dfn-content), border, and component-specific background. It does not include shadow and glow effects outside the component's content, background, or border.
NOTE 2
Examples of sub-components that may receive a focus indicator are menu items in an opened drop-down menu, or focusable cells in a grid.
NOTE 3
Contrast calculations can be based on colors defined within the [technology](https://www.w3.org/TR/WCAG22/#dfn-technologies) (such as HTML, CSS and SVG). Pixels modified by user agent resolution enhancements and anti-aliasing can be ignored.

link to understanding: https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html

**Comment by alexisreytech** [comment]
@rachaelbradley, thank you for this information. Currently on the website, our focus indicator is 3px and dashed. Would it be preferable to change the focus ring to be solid so it meets the requirements for [perimeter](https://www.w3.org/TR/WCAG22/#dfn-perimeter)?
